### DISCUSSION TITLE
Funktioniert leider nicht richtig bei mir

### DISCUSSION THREAD

**Discussion by halmich** [Trunks1982/Daikin-Rotex-HPSU-CAN#51] [Q&A]
Hallo,
ich habe auf mein ESP32 Board mit esphome Daikin-Rotex-HPSU-Can geflasht, alles an die Daikin Wärmepumpe angeschlossen, aber so richtig funktioniert es noch nicht.
Ich bekomme beispielsweise nicht alle Messwerte.
Jedoch kann ich z.B. vom Sommer Programm ins Kühlenprogramm wechseln.
Hat jemand eine Idee, woran das liegen kann?

![grafik](https://github.com/user-attachments/assets/a9329b32-072a-47f2-a665-f0483f9c3369)
![grafik](https://github.com/user-attachments/assets/8afda1ba-c949-4754-8952-199263a2d2e5)

**Comment by Trunks1982** [comment]
https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN

Unter Features!😉

**Reply by halmich** [reply]
sorry, ich verstehe deine Antwort nicht.
Ich kenne die Features, verstehe aber nicht, wieso manche funktionieren, vieles jedoch noch nicht.

**Comment by Trunks1982** [comment]
Abfrage Intervall🫣

**Reply by ap20132022** [reply]
> Abfrage Intervall🫣

was hältst du davon, wenn wir im Code diese Zeile hinzufüge:
restore_mode: ALWAYS_ON
--> Nach Neustart, Update wird die "intervall Abfrage" standardmäßig aktiviert. 
Darüber hinaus hab' ich die WW Temperatur als "initial Wert": 35°C geändert. Das verhindert, dass meine WP nach Neustart ungewollt WW kochen.
Ich hab' es bei mir so gemacht und läuft gut. 
@halmich : bin auch in die Falle gegangen.

**Reply by Trunks1982** [reply]
Das sind Standard Werte die ich benutze.Kann gerne jeder für sich selber ändern. In der regel startet man den ESP einmal und er läuft.
Man brauch ihn ja nicht so oft neustarten.

**Comment by halmich** [comment]
oh... ich verstehe.
it's not a bug 🙈
